tensorflow_datasets/core/dataset_info.py

```diff
@@ -58,6 +59,8 @@
 from tensorflow_metadata.proto.v0 import schema_pb2
 from tensorflow_metadata.proto.v0 import statistics_pb2
 
+import tensorflow_datasets.public_api as tfds
```

Comment:
This creates circular dependency (A import B, B import A). You should import individual files instead.

Reply:
I am doing
```python
from tensorflow_datasets.core import lazy_imports


plt = lazy_imports.matplotlib.pyplot 
```
but getting error - 
```
AttributeError: module 'tensorflow_datasets.core.lazy_imports' has no attribute 'matplotlib'
```

Reply:
When you get an error like this, it is good to search for examples of usage of this feature to make sure that you're using it correctly. In this case, by searching [usage of lazy_imports](https://github.com/tensorflow/datasets/search?q=%22from+tensorflow_datasets.core+import+lazy_imports%22&unscoped_q=%22from+tensorflow_datasets.core+import+lazy_imports%22), you would have noticed that `tensorflow_datasets.core.lazy_imports` is the file, while you want to access the `lazy_imports` object present in this file.

```
from tensorflow_datasets.core import lazy_imports
plt = lazy_imports.lazy_imports.matplotlib.pyplot 
```